src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -240,7 +241,7 @@ class InstrumentCodeAction implements Action<Task> {
             }
 
             ant.javac(source: getSourceCompatibility(), target: getTargetCompatibility(), encoding: getEncoding(),
-                      executable: getExecutable())
```

Comment:
In case you are wondering the groovyc task ignores the nested javac executable parameter. It gives a warning stating this fact. I removed the assignment since it was ignored to stop the warning.